spec/registry.md

```diff
@@ -885,14 +885,19 @@ an implementation-defined date/time type
 or a _date/time literal value_, as defined below.
 All other _operand_ values produce an _Invalid Expression_ error.
 
-A **_<dfn>date/time literal value</dfn>_** is a non-empty string consisting of 
-one of the following:
-- an XMLSchema 1.1 [dateTime](https://www.w3.org/TR/xmlschema11-2/#dateTime)
-- an XMLSchema 1.1 [time](https://www.w3.org/TR/xmlschema11-2/#time)
-- an XMLSchema 1.1 [date](https://www.w3.org/TR/xmlschema11-2/#date)
-
-The `timezoneOffset` of each of these formats is optional. 
-When the offset is not present, implementations should use a floating time type
+A **_<dfn>date/time literal value</dfn>_** is a non-empty string consisting of an ISO 8601 date,
+or an ISO 8601 datetime optionally followed by a timezone offset.
```

Comment:
Issue: why only a timezone offset? In general IANA time zone annotations should be used instead.

Did you consider citing IXDTF?